azure-core/azure/core/pipeline/transport/aiohttp.py

```diff
@@ -131,7 +128,7 @@ def _get_request_data(self, request):
         return response
 
 
-class AioHttpStreamDownloadGenerator(AsyncIterator):
+class AioHttpStreamDownloadGenerator:
```

Comment:
Not sure why you made this change? I think this base class in necessary.

Reply:
I believe AsyncIterator is not a class.